pkg/leasemanager/manager.go

```diff
@@ -0,0 +1,24 @@
+package leasemanager
```

Comment:
ditto.  I don't think this needs to exist.

Reply:
fixed